pontoon/base/views.py

```diff
@@ -740,27 +741,32 @@ def perform_checks(request):
 @transaction.atomic
 def download_translations(request):
     """Download translated resource."""
+
+    # FIXME: zip downloads should only be for projects with 2..10 resources
+    from pontoon.sync.utils import download_translations_zip
```

Comment:
This is a compromise reached after discussion with @mathjazz elsewhere. We'll be refactoring this code again as a part of the subsequent data model refactor, at which point we'll be able to introduce download support that won't require us to re-fetch any files.